spec/paper_trail/association_reify_error_behaviour/error_spec.rb

```diff
@@ -29,7 +29,7 @@
       expect {
         person.reload.versions.second.reify(has_one: true)
       }.to(
-        raise_error(::PaperTrail::Reifiers::HasOne::FoundMoreThanOne) do |err|
+        raise_error(::PaperTrailAssociationTracking::Reifiers::HasOne::FoundMoreThanOne) do |err|
```

Comment:
If you feel like it, please move this to a separate PR so it can be merged immediately.

Reply:
Here we go - https://github.com/paper-trail-gem/paper_trail/pull/1190